pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -56,6 +61,7 @@ func NewCmdIPFailoverConfig(f *clientcmd.Factory, parentName, name string, out,
 			ErrOut: errout,
 		},
 		ImageTemplate:    variable.NewDefaultImageTemplate(),
+		ServiceAccount:   "ipfailover",
```

Comment:
do we document that this has to be created, or do we want this command to create the named service account?